### ISSUE TITLE
Fix CSS for dark/light mode to conform to accessibility/contrast guidelines

### ISSUE THREAD

**Issue by srvanderplas** [srvanderplas/stat-computing-r-python#17]
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.88:1. Recommendation: change text colour to #000201.
Success Criterion:
[1.4.3: Contrast (Minimum)](http://www.w3.org/TR/WCAG21/#contrast-minimum)
Suggested Techniques:
[G18](https://www.w3.org/WAI/WCAG21/Techniques/general/G18)
Code Snippet 
...llout-icon no-icon"></i> </div><div class="callout-caption-container flex-fill"> Demonstrations </div>
